### PR TITLE
Bug 1747779 added note regarding F5 support

### PR DESCRIPTION
**Pull request by sheriff-rh**
Bug: 
https://bugzilla.redhat.com/show_bug.cgi?id=1747779
@openshift/team-documentation  please peer review :) 
@lihongan  please take a look.

### CODE REVIEW

**Review by bergerhoffer — COMMENTED** [review]
LGTM

File: release_notes/ocp-4-1-release-notes.adoc [hunk 1]
```diff
@@ -395,6 +395,13 @@ SR-IOV hardware on {product-title} nodes, which enables the user to
 attach SR-IOV virtual function (VF) interfaces to Pods in addition to other
 network interfaces.
 
+[id="ocp-4-1-f5"]
+==== F5 support
```
Comment (lihongan) [inline release_notes/ocp-4-1-release-notes.adoc]: Just like 3.11 doc, use `F5 router plug-in support` is better.

File: release_notes/ocp-4-1-release-notes.adoc [hunk 2]
```diff
@@ -395,6 +395,13 @@ SR-IOV hardware on {product-title} nodes, which enables the user to
...
+
+F5 integration is no longer supported as part of {product-title} directly.
```
Comment (lihongan) [inline release_notes/ocp-4-1-release-notes.adoc]: Same here, use `F5 router plug-in is no longer supported...`

**Comment by lihongan** [comment]
Other looks good, thank you.

File: release_notes/ocp-4-1-release-notes.adoc (hunk 1 above)
Comment (sheriff-rh) [reply release_notes/ocp-4-1-release-notes.adoc]: I agree. I have committed the changes.

File: release_notes/ocp-4-1-release-notes.adoc (hunk 2 above)
Comment (sheriff-rh) [reply release_notes/ocp-4-1-release-notes.adoc]: I agree. I have committed the changes.

**Pull request merged**